packages/styles/scss/patterns/listsection/index.scss

```diff
@@ -3,10 +3,6 @@
 @import './listsection-group';
 @import './listsection-item';
 
-.#{$prefix}--listsection--g100 {
-  @include carbon--theme($carbon--theme--g100) {
-    @include listsection;
-    @include listsection-group;
-    @include listsection-item;
-  }
-}
+@include listsection;
+@include listsection-group;
+@include listsection-item;
```

Comment:
I don't think these are necessary since they are already getting exported in the files themselves...

Reply:
donee

Reply:
donee